beebot/body/body.py

```diff
@@ -116,6 +117,13 @@ def cycle(self) -> Memory:
         self.execute(decision=self.decide())
 
         complete_memory = self.current_memory_chain.finish()
+
+        baserun.log("CycleComplete", payload={
```

Comment:
This is a noop if baserun is not initialized. We are automatically tracking OpenAI calls but I remember hearing that it's also useful to track aggregated info after every cycle, so added this as one option